ngraph/core/reference/include/ngraph/runtime/reference/normalize_l2.hpp

```diff
@@ -0,0 +1,69 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            template <typename T>
+            void normalize_l2(const T* data,
+                              T* out,
+                              const Shape& data_shape,
+                              const AxisSet& reduction_axes,
+                              float eps,
+                              op::EpsMode eps_mode)
+            {
+                AxisSet axes = reduction_axes;
```

Comment:
@ilyachur @GlebKazantaev Could you please review NormalizeL2 reference implementation?

Reply:
@ilyachur @GlebKazantaev ping